Add blog post on Spring Native + Okta Spring Boot starter

mraible opened the pull request:
This blog post has the following features:

- [x] A title that's approved by Dev Advocacy
- [x] A URL that's approved by Dev Advocacy
- [x] The content has been run through Grammarly (https://www.grammarly.com/)
- [x] Rendered locally and confirmed that no Markdown typos exist

mraible committed:
Add blog post on Spring Native + Okta Spring Boot starter

mraible committed:
Fix quotes

mraible committed:
Polishing

mraible committed:
Add State of Spring reference

mraible committed:
Fix image and add TOC

mraible committed:
Clarify

mraible committed:
Add link to SpringOne presentation

mraible committed:
Add link to Okta starter release

mraible committed:
Formatting

mraible committed:
Back to 2 lines

mraible committed:
Make more fun

mraible committed:
AsciiDoc, not HTML!

mraible committed:
Add link to CLI video

havi commented on _source/_posts/2021-09-16-spring-native-okta-starter.adoc:
```diff
@@ -0,0 +1,179 @@
+
+== Okta Spring Boot Starter v2.2 Supports Spring Native!
+
+Today, I'm happy to announce that we've added our learnings to https://github.com/okta/okta-spring-boot/releases/tag/okta-spring-boot-parent-2.2.0[Okta Spring Boot v2.2.0]; now, it only takes a couple of lines to add native support to a Spring Boot application that uses Okta.
```
The URL <https://github.com/okta/okta-spring-boot/releases/tag/okta-spring-boot-parent-2.2.0> 404s. Not sure what the correct URL is

mraible replied:
This is expected. The release hasn't happened yet. :)

havi replied:
That's good to know! :) Any chance of publishing the blog post today? (I guess it will depend on pending release...)

mraible replied:
We don't want to publish it today, we want to publish it with the video tomorrow. I still need to create the video's thumbnail.

havi reviewed (commented):
Once that 404 GH URL is fixed, this is approved to ship. Thanks for pulling it together so quickly.

mraible committed:
Add tweets

2nd tweet has special @ from https://twitter.com/kaqqao/status/1437348024599162880

bdemers commented on _source/_posts/2021-09-16-spring-native-okta-starter.adoc:
```diff
@@ -0,0 +1,179 @@
+        </snapshots>
+    </repository>
+</repositories>
+<pluginRepositories>
+    <pluginRepository>
+        <id>spring-releases</id>
+        <name>Spring Releases</name>
+        <url>https://repo.spring.io/release</url>
+        <snapshots>
+            <enabled>false</enabled>
+        </snapshots>
+    </pluginRepository>
+</pluginRepositories>
```
```suggestion
```

nit: I'm guessing you only need the `repository` for the aot dep.  

That said, this doesn't hurt, and any time you want to bump a Spring Boot version you want to include the `pluginRepository` anyway (to get the newer `spring-boot-maven-plugin`)

mraible replied:
I tried removing both and it fails with:

```
[ERROR] Plugin org.springframework.experimental:spring-aot-maven-plugin:0.10.3 or one of its dependencies could
 not be resolved: Could not find artifact org.springframework.experimental:spring-aot-maven-plugin:jar:0.10.3 in 
 central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

If I add `<pluginRepositories>` only, it fails too:

```
[ERROR] Failed to execute goal on project demo: Could not resolve dependencies for project 
 com.example:demo:jar:0.0.1-SNAPSHOT: Could not find artifact 
 org.springframework.experimental:spring-native:jar:0.10.3 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

So, it seems both are needed.

bdemers replied:
Doh, sorry, I see it's a plugin 🤦 

Sorry for the noise

mraible committed:
Add social image and optimize images

mraible committed:
Match case of State of Spring

mraible committed:
Crop State of Spring slide

mraible commented on _source/_posts/2021-09-16-spring-native-okta-starter.adoc:
```diff
@@ -0,0 +1,179 @@
+
+== Okta Spring Boot Starter v2.2 Supports Spring Native!
+
+Today, I'm happy to announce that we've added our learnings to https://github.com/okta/okta-spring-boot/releases/tag/okta-spring-boot-parent-2.2.0[Okta Spring Boot v2.2.0]; now, it only takes a couple of lines to add native support to a Spring Boot application that uses Okta.
```
```suggestion
Today, I'm happy to announce that we've added our learnings to https://github.com/okta/okta-spring-boot/releases/tag/okta-spring-boot-parent-2.1.1[Okta Spring Boot v2.1.1]; now, it only takes a couple of lines to add native support to a Spring Boot application that uses Okta.
```

mraible committed:
Fix version number

mraible committed:
Change SpringOne link to more popular tweet

mraible committed:
More than 90 minutes

mraible committed:
Merge branch 'main' into spring-native-okta-starter
